notification/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * If you are deploying extension module as a serverless function,
+ * this will be the main javascript file.
+ *
+ * This function has tested as Google Cloud Function
+ *
+ * Entry point: notificationTrigger
+ */
+
+const notificationController = require('./src/api/notification/notification.controller')
+const httpUtils = require('./src/utils/commons')
+
+httpUtils.collectRequestData = (request) => JSON.stringify(request.body)
```

Comment:
Same comment as for extension would be nice :) and also try to have lambda in extension index.js as you have it here.

Reply:
The implementation has been updated.